instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integrationtests/utils/MockServerRule.kt

```diff
@@ -57,4 +66,21 @@ internal class MockServerRule<T : Activity>(activityClass: Class<T>) :
             .setResponseCode(code)
             .setBody("{}")
     }
+
+    private fun RecordedRequest.unzip(): String {
```

Comment:
Nice job Marius 😄

Reply:
Actually I thought would be better to keep the compression for our tests also, this will allow to test in a real scenario.